apps/miniapp/domains/common/utils/oidcAuth.tsx

```diff
@@ -40,7 +51,54 @@ const OidcAuthProvider = ({ children }) => {
             }
         }
 
-    }, [isAllLoaded, user, isAuthenticated, condoUser, condoOrganization])
+    }, [isAllLoaded, user, isAuthenticated, condoUser, condoOrganization, hasStorageAccess])
+
+    useEffect(() => {
+        if (!hasStorageAccess && document && document.hasStorageAccess) {
+            document.hasStorageAccess().then(
+                (hasAccess) => {
+                    // Boolean hasAccess says whether the document has access or not.
+                    console.debug(`document.hasStorageAccess() resolved: ${hasAccess}`)
```

Comment:
Debug?

Reply:
Yes, because this is really a debug message. So I'd like to hide them from info messages within browser console.